src/styles/Button.js

```diff
@@ -1,5 +1,11 @@
 import styled from 'styled-components'
 
+export const FormButtonSet = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+`
+
```

Comment:
I think it's good to have it abstracted since it's being used for all form, but maybe could move to another file more specific to forms? Let me know what you think, or if I'm misunderstanding.